treeherder/log_parser/crossreference.py

```diff
@@ -63,6 +64,8 @@ def _crossreference(job):
                 summary=summary,
                 line_number=error.line_number,
                 failure_line=failure_line))
+            error.failure_line = failure_line
+            error.save()
```

Comment:
You could use update_fields to make this a little faster.

https://docs.djangoproject.com/en/1.10/ref/models/instances/#specifying-which-fields-to-save